RubberduckTests/Symbols/DeclarationFinderTests.cs

```diff
@@ -1,4 +1,5 @@
 
+using System.CodeDom;
```

Comment:
Do we actually use this? If yes, it is OK.

Reply:
No it wasn't. Removed.